<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontPropertiesDialog</class>
 <widget class="QDialog" name="FontPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font Properties</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/qtgameengine/icons/font-file.png</normaloff>:/qtgameengine/icons/font-file.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName">
       <property name="toolTip">
        <string>The name of the font</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelFont">
       <property name="text">
        <string>Font:</string>
       </property>
       <property name="buddy">
        <cstring>fontComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxSize</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QFontComboBox" name="fontComboBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxSize"/>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxBold">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxItalic">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCharacterRange">
     <property name="title">
      <string>Character Range</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="labelTill">
        <property name="text">
         <string>till</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBoxCharRangeFrom">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spinBoxCharRangeTo">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>127</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonNormal">
        <property name="toolTip">
         <string>Set the character range to the normal range (32 - 127)</string>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonDigits">
        <property name="toolTip">
         <string>Set the character range to the digits range (48 - 57)</string>
        </property>
        <property name="text">
         <string>Digits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonAll">
        <property name="toolTip">
         <string>Set the character range to the complete set (0 - 255)</string>
        </property>
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonLetters">
        <property name="toolTip">
         <string>Set the character range to the letters range (65 - 122)</string>
        </property>
        <property name="text">
         <string>Letters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPreview">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>AaBbCcDd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FontPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FontPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
